tests/dummy/app/pods/docs/index/template.md

```diff
@@ -1,13 +1,28 @@
-# Ember Table
+<div class="docs-hero">
+  {{docs-hero
+    logo='ember'
+    slimHeading='table'
+    byline='the power table for power users'
+  }}
+</div>
 
-An addon to support large data set and a number of features around table. Ember Table can handle over 100,000 rows without and rendering or performance issue. This version of Ember Table supports Ember 1.11 to latest version of Ember.
+Ember Table is a power table made for users who need a full-fledged,
```

Comment:
are we not too "elitist" here?  it seems like that it works pretty well out of the box, no?

Reply:
Not sure what you mean exactly here, can you clarify?